Makefile

```diff
@@ -127,6 +127,20 @@ vet: gowork ## Run go vet against code.
 	go vet ./...
 	go vet ./api/...
 
+APIPATH ?= $(shell pwd)/api
+.PHONY: tidy
+tidy: fmt
+	go mod tidy; \
+	pushd $(APIPATH); \
+	go mod tidy; \
+	popd
```

Comment:
As I noted in a previous comment, you don't need to pushd/popd. I'm also not sure why you use "; \" because that's essentially what 'make' does when it processes the commands.

This should work:

```
.PHONY: tidy
tidy: fmt
	go mod tidy
	cd ./api
	go mod tidy
```

Reply:
Hi ASBishop, I try to edit this pre-commit change like the nova-operator did here https://github.com/openstack-k8s-operators/nova-operator/pull/313  "Add revive and ginkgolinter to golanci"

Reply:
Please don't resolve conversations this way, it simply hides reviewer feedback.

The other Makefile (nova's) is also wrong. Repeating a bad pattern just for consistency is not a good idea.